scripts/partition_manager.py

```diff
@@ -193,16 +199,9 @@ def set_sub_partition_address_and_size(reqs, sub_partitions):
         if size == 0:
             raise RuntimeError("No compatible parent partition found for %s" % sp_name)
         address = min([reqs[part]['address'] for part in sp_value['span']])
```

Comment:
Should we remove the "span" functionality?

Reply:
As discussed f2f: 'sub_partitions' are removed in a later commit. 'span' will stay since it still has uses.